### PR TITLE
Add Traditional Chinese Localization

### PR DESCRIPTION
**Pull request by ethanhuang13**
I've move all strings from view controller files to Localizable.strings, then add Traditional Chinese localization.

### CODE REVIEW

File: App/WWDC Family/View Controllers/Settings/Fuzzing/FuzzingViewController.m [hunk 1]
```diff
@@ -79,10 +79,10 @@ - (instancetype)init {
     
     // Radius
     ASTextNode *radiusTitleNode = [ASTextNode new];
-    [radiusTitleNode setAttributedText:[[NSAttributedString alloc] initWithString:@"Radius" attributes:@{NSFontAttributeName:[UIFont systemFontOfSize:15 weight:UIFontWeightHeavy], NSForegroundColorAttributeName:[UIColor colorWithWhite:0.0 alpha:0.6]}]];
+    [radiusTitleNode setAttributedText:[[NSAttributedString alloc] initWithString:NSLocalizedString(@"", @"Radius") attributes:@{NSFontAttributeName:[UIFont systemFontOfSize:15 weight:UIFontWeightHeavy], NSForegroundColorAttributeName:[UIColor colorWithWhite:0.0 alpha:0.6]}]];
```
Comment (ay8s) [inline App/WWDC Family/View Controllers/Settings/Fuzzing/FuzzingViewController.m]: Looks like we're missing this string.

File: App/WWDC Family/View Controllers/Settings/SettingsViewController.m [hunk 2]
```diff
@@ -239,7 +239,7 @@ - (void)tableNode:(ASTableNode *)tableNode didSelectRowAtIndexPath:(NSIndexPath
         }
     } else if(indexPath.section == 5){
         if(indexPath.row == 0){
-            UIAlertController *alert = [UIAlertController alertControllerWithTitle:@"About" message:@"Building upon past WWDC Family apps, ConfFriends was built for privacy and performance in mind by Andrew Yates (@ay8s) using Texture & Firebase, inspired by Felix Krause's (@KrauseFx) past iteration.\n\nConfFriends will be available on GitHub shortly." preferredStyle:UIAlertControllerStyleAlert];
+            UIAlertController *alert = [UIAlertController alertControllerWithTitle:NSLocalizedString(@"settingsVC.alert.about", @"About") message:NSLocalizedString(@"", @"Building upon past WWDC Family apps, ConfFriends was built for privacy and performance in mind by Andrew Yates (@ay8s) using Texture & Firebase, inspired by Felix Krause's (@KrauseFx) past iteration.\n\nConfFriends will be available on GitHub shortly.") preferredStyle:UIAlertControllerStyleAlert];
```
Comment (ay8s) [inline App/WWDC Family/View Controllers/Settings/SettingsViewController.m]: Missing a key for the about message also.

File: App/WWDC Family/View Controllers/Settings/Fuzzing/FuzzingViewController.m (hunk 1 above)
Comment (ethanhuang13) [reply App/WWDC Family/View Controllers/Settings/Fuzzing/FuzzingViewController.m]: Good catch. Updated.

File: App/WWDC Family/View Controllers/Settings/SettingsViewController.m (hunk 2 above)
Comment (ethanhuang13) [reply App/WWDC Family/View Controllers/Settings/SettingsViewController.m]: Updated. I also removed 'ConfFriends will be available on GitHub shortly' in the about message.

**Pull request merged**